CHANGELOG-1.x.md

```diff
@@ -1,3 +1,14 @@
+# v1.1.0
+
+## Notable changes
+- The hostPath directory where the driver DaemonSet Pods write state files to their respective Node hosts has changed to fix the driver not working on Bottlerocket OS Nodes. No matter what OS your Nodes are running, you must use a supported method like helm or kustomize to update the driver. If not, i.e. if you only change the image tag of your DaemonSet, the migration from old to new directory won't succeed. See "change config dir location" below for details.
```

Comment:
Release notes were trimmed to remove all helm chart related changes which technically were already released in helm chart 1.0.0 last week.

And then I added this warning about not using helm/kustomize.